.github/workflows/pypi-whl.yml

```
@@ -0,0 +1,96 @@
+name: PyPI Whl Publish
+
+on:
+  push:
+    branches:
+      - main
+      - deno-binary
```

overall here aim for the same setup of `on` as in https://github.com/bids-standard/bids-validator/blob/main/.github/workflows/deno_tests.yml

```suggestion
on:
  push:
    branches: [main, dev]
    tags: ['*']
  pull_request:
    branches: [main, dev]
```

and then add corresponding `if` below for `twine` call: